maths/sum_of_digits.py

```diff
@@ -0,0 +1,18 @@
+def digit_sum(n: int) -> int:
```

Comment:
```suggestion
def sum_of_digits(n: int) -> int:
```
Seems like a more self documenting name.